image_data_guide/01_download_data.ipynb

```diff
@@ -0,0 +1,546 @@
+      "text/plain": [
+       "<Figure size 720x504 with 12 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, axs = plt.subplots(3, 4, figsize=(10, 7))\n",
+    "indices = np.random.randint(low=0, high=249, size=12)\n",
+    "\n",
+    "for i, ax in enumerate(axs.flatten()):\n",
+    "    ax.imshow(frog_images[indices[i]])\n",
+    "    ax.axis('off')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Write Sample Frog Images to `data_sample_2500` Directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "frog_filenames = np.array(batch_1[b'filenames'])[sample_frog_indices]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for idx, filename in enumerate(frog_filenames):\n",
+    "    filename = filename.decode()\n",
+    "    data = frog_images[idx]\n",
+    "    if filename.endswith('.png'):\n",
+    "        filename = filename.replace('.png', '.jpg')\n",
+    "    imwrite(sample_dir/filename, data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample_dir.rename('data_sample_2750')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add Frog Annotations to `sample_annos`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "category_labels[26] = 'frog'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "next_anno_idx = np.array(list(sample_annos.keys())).max() + 1\n",
+    "\n",
+    "frog_anno_ids = range(next_anno_idx, next_anno_idx + len(frog_images))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for idx, frog_id in enumerate(frog_anno_ids):\n",
+    "    sample_annos[frog_id] = {'category_id': 26, 'file_name': frog_filenames[idx].decode().replace('.png', '.jpg')}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Store Annotations for Next Guides"
```

Comment:
Describe what the purpose of doing this is for. e.g. "Now that you downloaded the dataset, store the annotations so you can use them in the next notebook".

Reply:
Added reason for persisting data